.github/workflows/build_ubuntu.yaml

```diff
@@ -41,6 +40,9 @@ jobs:
 
     - name: Install dependencies
       run: |
+        if [[ ${{ matrix.mpi }} == "openmpi" ]]; then
+          echo "$HOME/mpi/bin" >> $GITHUB_PATH
+        fi
```

Comment:
looks right. 
I forget now, is there no openmpi packages on Ubuntu?

Reply:
There is, but when using that you get undefined references to functions from the MPI C++ library (from ESMF) when linking executables using the MPI Fortran wrapper. If you build it yourself you don't get that.

I've seen this in other repos (UFS_UTILS). I think it's related to how Ubuntu builds the binaries, but I'm not exactly sure.

Reply:
Thanks @kgerheiser 
[This](https://www.rosettacommons.org/node/1850) post seems to suggest that there is a solution.  May be worth a try in  the future and save us some compile time in the action.

Reply:
It's cached so it doesn't build it every time, but I will look at that.